pkg/movies/handlers.go

```diff
@@ -13,11 +13,39 @@ type handler struct {
 	app application.App
 }
 
+func (h *handler) createHandler(c echo.Context) error {
+	// params is a struct that will have our payload bound and validated against
+	params := createParams{}
+	if err := c.Bind(&params); err != nil {
+		// if there is an error binding or validating the payload, return early with an error
+		return err
+	}
+
+	movie := model.Movie{
+		Title:       params.Title,
+		ReleaseDate: params.ReleaseDate,
+	}
+
+	_, err := h.app.DB.Model(&movie).Insert()
```

Comment:
Just to call out an alternative preference -- something I prefer when writing Go handlers is to, instead of attaching the whole `application.App` struct to the `handler`, explicitly set the fields from `application.App` that you want to use. So in looking at the handlers, you know exactly what they'll be accessing in the handler functions. In this case we'd add a `db` field to the `handler` struct and just access it through `h.db`.

Reply:
I prefer that way as well but I was just following the Goyagi approach here. I'm also building Goyagi in accordance with the Go clean architecture. I'll be curious to know your thoughts on that once it is done.